### ISSUE TITLE
Gluon: Per-layer learning rate for fine tuning a pretrained network

### ISSUE THREAD

**Issue by JWarlock** [apache/incubator-mxnet#13391]
Hi!
I want to train a pretrained model, for example, ResNet, where I want to limit the first two layers’ learning rate to lr/100, middle layers ’ learning rate to lr/10, and fully connected layers ’ learning rate to lr.
I’ve only found implementation using native mxnet: https://github.com/apache/incubator-mxnet/issues/2242.
But How to implement it using gluon and gluon.Trainer?

**Comment by vdantu** [comment]
@mxnet-label-bot add [gluon, question]

**Comment by eric-haibin-lin** [comment]
You can set the lr_mult attribute: 
```
net.xx_layer.collect_params().setattr('lr_mult', 10)
```

**Comment by piyushghai** [comment]
https://discuss.mxnet.io/t/gluon-per-layer-learning-rate-for-fine-tuning-a-pretrained-network/2288

This question got answered here. I think this is good to close.

**Comment by eric-haibin-lin** [comment]
Closing this for now. Feel free to comment if not resolved
